hydra/controllers/client.py

```diff
@@ -816,3 +817,67 @@ def disable_metrics(self):
 
         self.restart_service()
         self.app.log.info(f'Successfully disabled all metrics reporting.')
+
+    @ex(
+        arguments=[
+            (
+                ['-n', '--name'],
+                {
+                    'help': 'name of network to join',
+                    'action': 'store',
+                    'dest': 'name'
+                }
+            ),
+            (
+                ['-v', '--version'],
+                {
+                    'help': 'version of binary',
+                    'action': 'store',
+                    'dest': 'version'
+                }
+            ),
+            (
+                ['-D', '--destroy'],
```

Comment:
Actually, I think the upgrade command should always upgrade the binary, regardless of parameters. It might be a good idea to add a prompt (https://cement.readthedocs.io/en/3.0/api/utils/shell/#cement.utils.shell.Prompt) during the command, asking the users to confirm the upgrade, and confirm the version.

Reply:
@kevingduck I don't think we need this parameter anymore right?

Reply:
SGTM, removed --version parameter.